db/warehouse_structure.sql

```diff
@@ -150,8 +150,8 @@ CREATE TYPE public.record_type AS ENUM (
 
 CREATE FUNCTION public.f_unaccent(text) RETURNS text
     LANGUAGE sql IMMUTABLE STRICT PARALLEL SAFE
-    AS $_$
-SELECT public.unaccent('public.unaccent', $1)  -- schema-qualify function and dictionary
+    AS $_$
```

Comment:
These changes are unrelated, I was trying to figure out how to revert them but they aren't showing up as changes at all in my local ide! Anyway, I think nothing is really changing here and not sure why github detects it as changed.

Reply:
It shouldn't be anything to worry about as long as the tests are still passing. We all have minor differences in this file and it gets formatted differently if you open it in an ide, etc.